crates/sncast/src/starknet_commands/account/mod.rs

```diff
@@ -162,11 +161,24 @@ pub fn add_created_profile_to_configuration(
 #[cfg(test)]
 mod tests {
     use camino::Utf8PathBuf;
-    use sncast::helpers::configuration::{copy_config_to_tempdir, CastConfig};
+    use configuration::CONFIG_FILENAME;
+    use sncast::helpers::configuration::CastConfig;
     use sncast::helpers::constants::DEFAULT_ACCOUNTS_FILE;
     use std::fs;
+    use tempfile::{tempdir, TempDir};
 
     use crate::starknet_commands::account::add_created_profile_to_configuration;
+    #[must_use]
+    pub fn copy_config_to_tempdir(src_path: &str, additional_path: Option<&str>) -> TempDir {
```

Comment:
this is the same function that is defined in configuration crate, let's just import it :+1:

Reply:
Done